commonmark-ext-gfm-strikethrough/src/main/java/org/commonmark/ext/gfm/strikethrough/internal/StrikethroughNodeRenderer.java

```diff
@@ -0,0 +1,44 @@
+package org.commonmark.ext.gfm.strikethrough.internal;
+
+import org.commonmark.ext.gfm.strikethrough.Strikethrough;
+import org.commonmark.html.HtmlWriter;
+import org.commonmark.html.renderer.NodeRenderer;
+import org.commonmark.html.renderer.NodeRendererContext;
+import org.commonmark.node.Node;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.Set;
+
+public class StrikethroughNodeRenderer implements NodeRenderer {
```

Comment:
Why does the `CoreNodeRenderer` extend `AbstractVisitor` but the extension renderers do not? `renderChildren()` here is reimplmenting `visitChildren()`.